demo/demo-elements.js

```diff
@@ -0,0 +1,103 @@
+import '@polymer/polymer/polymer-legacy.js';
+import '../google-youtube.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<dom-module id="demo-element">
+  <template id="page-template">
+    <style>
+      div {
+        margin-bottom: 1em;
+      }
+    </style>
+    <h1><code>&lt;google-youtube&gt;</code> Demo</h1>
+    <h3>Full API Demo</h3>
+    <google-youtube id="googleYouTube" playsupported="{{playSupported}}" video-id="mN7IAaRdi_k" state="{{state}}" currenttime="{{currentTime}}" currenttimeformatted="{{currentTimeFormatted}}" duration="{{duration}}" durationformatted="{{durationFormatted}}" fractionloaded="{{fractionLoaded}}" volume="{{volume}}" playbackrate="{{playbackRate}}" playbackquality="{{playbackQuality}}" on-google-youtube-state-change="handleStateChange" on-google-youtube-error="handleYouTubeError">
```

Comment:
This is a demo, can you please beautify the attributes on this file?

Reply:
Done. Wrapped lines longer than 100 characters.